XCode/Tests/XCTestManifests.swift

```diff
@@ -97,6 +97,7 @@ extension SwifterTestsWebSocketSession {
 
 public func __allTests() -> [XCTestCaseEntry] {
     return [
+        testCase(IOSafetyTests.__allTests__IOSafetyTests),
```

Comment:
Can you please remove the entry for `testCase(IOSafetyTests.__allTests__IOSafetyTests)`. Danger should be alerted you but it disappears once your update it the PR. For more reference see [#366](https://github.com/httpswift/swifter/issues/366).